ci/openstack/vars.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+if [ "${RUN_OPENSTACK_CI:-}" == true ]
+then
+    # TODO(shadower): Can we only run this when the project admin asked for it?
```

Comment:
Yeah but I'm not sure how to integrate this here. Travis doesn't let you run builds manually (as far as I could figure out).

The only thing I can think of is: the first build would always fail (because there are no comments), but then someone could say "ci check this" and manually restart the travis build.

That seems cumbersome to me (but maybe is worth it?)